liblangutil/EVMVersion.h

```diff
@@ -87,6 +89,7 @@ class EVMVersion:
 	bool hasExtCodeHash() const { return *this >= constantinople(); }
 	bool hasChainID() const { return *this >= istanbul(); }
 	bool hasSelfBalance() const { return *this >= istanbul(); }
+	bool hasBaseFee() const { return *this >= london(); }
 
 	bool hasOpcode(evmasm::Instruction _opcode) const;
```

Comment:
Needs change

Reply:
It's done in https://github.com/ethereum/solidity/pull/11763/commits/0c8b6124e3a4cb0d42e9647f5ccddfd5e5474926

Reply:
Or second commit in https://github.com/ethereum/solidity/pull/11763